transport/rif_comms/cluster.py

```diff
@@ -0,0 +1,27 @@
+from raiden.tests.integration.network.transport.rif_comms.node import Node, Config
+
+
+class Cluster:
+    def __init__(self, nodes_to_clients: dict):
+        self.nodes_to_clients = nodes_to_clients
```

Comment:
for me—just a personal opinion—the phrase "nodes to clients" suggest a function or method.

i would name this `node_clients` or `client_topology` instead and explain this attribute through the code documentation

Reply:
nodes_to_clients represents the mapping for me, but node_clients too

changed.

Reply:
node_to_clients seems like a dict key_to_value, the docs are on the cluster class  so i think we can keep it like this

Reply:
yeah, we certainly can :+1: